content/chainguard/chainguard-images/how-to-use/verifying-chainguard-images-and-metadata-signatures-with-cosign/index.md

```diff
@@ -21,20 +21,57 @@ toc: true
+This guide outlines how you can use Cosign to verify container image signatures and attestations, as well as download container attestations. 
+
+## Prerequisites
+
+The following examples require [Cosign](/open-source/sigstore/cosign/how-to-install-cosign/) and [jq](https://stedolan.github.io/jq/) to be installed on your machine in order to download and verify image attestations. The examples in this guide will pull detailed information about all signatures found for the provided image.
 
 ## Registry and Tags for Chainguard Containers
 
 Attestations are provided per image build, so you'll need to specify the correct tag and registry when pulling attestations from an image with `cosign`.
 
-- `cgr.dev/chainguard` - the Public Registry contains our **Starter container images**, which typically comprise the `:latest` versions of an image.
-- `cgr.dev/your-registry-name` - A Private/Dedicated Registry contains our **Production container images**, which include all versioned tags of an image and special images that are not available in the public registry (including FIPS images and other custom builds).
+- `cgr.dev/chainguard`: The Public Registry contains Chainguard's **Starter container images**, which typically comprise the `:latest` versions of an image.
+- `cgr.dev/YOUR-ORGANIZATION`: A Private/Dedicated Registry contains your organization's **Production container images**, which include all versioned tags of an image and special images that are not available in the public registry (including FIPS images and other custom builds).
 
 The commands listed on this page will default to the `:latest` tag, but you can specify a different tag to fetch attestations for.
 
+
+## Chainguard's Signing Identities
+
+Chainguard uses an identity associated with its official GitHub account to sign images in the public registry that contains the free tier of Starter images.
+
+For private images, Chainguard signs all images in your private registry with one of two different identities in your organization:
+
+* The `catalog_syncer` identity is used to sign images that have been imported directly from the Chainguard Containers catalog. 
+* The `apko_builder` identity is used to sign any images that have been customized for your organization, such as those built with [Custom Assembly](/chainguard/chainguard-images/features/ca-docs/custom-assembly/).
+
+These identities are created and added to every [verified Chainguard organization](/chainguard/administration/iam-organizations/verified-orgs/) automatically. 
+
+To follow along with the **Private Registry** examples in this guide, you will need the UIDPs of these Chainguard identities. To this end, create a few environment variables, the first of which should point to the name of your Chainguard organization:
+
+```shell
+PARENT=your-organization
+```
+
+Next, use this variable to create two more to hold the UIDPs of your organization's `catalog_syncer` and `apko_builder` identities, respectively:
```

Comment:
Maybe "Next, create two more variables"? I see what you're saying, but it threw me for a second and I don't think you need to mention the previous variable